upgrades/2.4.22.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+use NewfoldLabs\WP\ModuleLoader\Container;
+
+add_action(
+	'newfold_container_set',
+	function ( Container $container ) {
+		nfd_update_options_table( $container );
```

Comment:
@manikantakailasa This is looking good. One last request. Let's remove the `nfd_update_options_table()` function and add the code inline here since it is only used once.